packages/clerk-js/src/ui/components/UserProfile/UserProfileRoutes.tsx

```diff
@@ -14,6 +15,12 @@ export const UserProfileRoutes = () => {
   const isSecurityPageRoot = pages.routes[0].id === USER_PROFILE_NAVBAR_ROUTE_ID.SECURITY;
   const isBillingPageRoot = pages.routes[0].id === USER_PROFILE_NAVBAR_ROUTE_ID.BILLING;
 
+  const BillingPage = lazy(() =>
+    import('./BillingPage').then(module => ({
```

Comment:
Let's track its size, and add it to bundlewatch config.

```suggestion
    import( /* webpackChunkName: "up:billing-page"*/ './BillingPage').then(module => ({
```

Reply:
Changed it [here](https://github.com/clerk/javascript/pull/5409/commits/a200ec61dc1eeea942f1512953a9f8158a98356f)